docs/user-guide/install-configure-zos-extensions.md

```diff
@@ -50,69 +32,43 @@ Execute the command from z/OS USS. Use the following command line parameters:
 
   (Boolean, Optional) Enable the most detail trace level logging. This will help on troubleshooting issues.
 
-**Examples:**
-
-The following command installs the `my-zowe-component-1.2.3.pax` into `/global/zowe/extensions` which is defined as `zowe.extensionDirectory` in `/path/to/my/zowe.yaml`.
-
-```
-$ zwe components install -o /path/to/my-zowe-component-1.2.3.pax -c /path/to/my/zowe.yaml
```

Comment:
I think this is a good example to keep, maybe in addition to what you added

Reply:
it's covered in the zwe help linked in the same page, so i removed it as a duplicate.